client/control/components/RecipeForm.js

```diff
@@ -37,14 +42,51 @@ export class RecipeForm extends React.Component {
       action: pt.string.isRequired,
       arguments: pt.object.isRequired,
     }),
-  }
+    // route prop passed from router
+    route: pt.object,
+  };
+
   static argumentsFields = {
     'console-log': ConsoleLogFields,
     'show-heartbeat': HeartbeatFields,
   };
 
+  renderCloningMessage() {
+    const isCloning = this.props.route && this.props.route.isCloning;
+    const { recipeId } = this.props;
+    const displayedRecipe = this.props.recipe || {};
+
+    return isCloning ?
+      (<span className="cloning-message">
+        You're cloning {displayedRecipe.name} ({displayedRecipe.action}).
```

Comment:
Not sure - I'm wondering if it's how I'm splitting up the lines. Easiest fix is to just switch to `You are`, that work?